msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -468,13 +472,10 @@ PUT {Service URL of your bot}/v3/conversations/{conversationId}/activities/{acti
 
 ## Get soft delete message event
 
-When you soft delete a message in a chat, the bot gets a notification of the soft delete message event.
+When you soft delete a message in the group chat or channel, the bot gets a notification of the soft delete message event.
```

Comment:
update it to "When you soft delete a message, the bot gets a notification of the soft delete message activity."

Reply:
updated as suggested